test/e2e/bandwidth_test.go

```diff
@@ -19,26 +19,31 @@ import (
 	"strconv"
 	"strings"
 	"testing"
+
+	v1 "k8s.io/api/core/v1"
 )
 
+const iperfPort = 5201
+
 // BenchmarkBandwidthIntraNode runs the benchmark of bandwidth between Pods on same node.
 func BenchmarkBandwidthIntraNode(b *testing.B) {
 	withPerfTestSetup(func(data *TestData) {
 		b.StartTimer()
 
-		podDefA := createPerfTestPodDefinition("perftest-a", perftoolContainerName, perftoolImage)
-		if _, err := data.clientset.CoreV1().Pods(testNamespace).Create(podDefA); err != nil {
-			b.Fatalf("Error when creating the first perftest Pod: %v", err)
+		if err := data.createPodOnNode("perftest-a", masterNodeName(), perftoolImage, nil, nil, nil, nil); err != nil {
+			b.Fatalf("Error when create the perftest client Pod: %v", err)
 		}
-		podDefB := createPerfTestPodDefinition("perftest-b", perftoolContainerName, perftoolImage)
-		if _, err := data.clientset.CoreV1().Pods(testNamespace).Create(podDefB); err != nil {
-			b.Fatalf("Error when creating the second perftest Pod: %v", err)
+		if err := data.podWaitForRunning(defaultTimeout, "perftest-a", testNamespace); err != nil {
+			b.Fatalf("Error when waiting the perftest client Pod: %v", err)
```

Comment:
"Error when waiting **for** the perftest client Pod"

Reply:
same below

Reply:
Done.